syntax = "proto2";

package birthday_push_controller;

message BirthInfo {
  optional string user_id = 1;
  optional string birth_date = 2;
  optional string wwid = 3;
}
